app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -272,8 +272,12 @@ private void updateUI() {
 
         mBinding.navigationBarNavigation.desktopModeButton.setOnClickListener(view -> {
             final int defaultUaMode = SettingsStore.getInstance(mAppContext).getUaMode();
-            mHamburgerMenu.setUAMode(defaultUaMode);
-            mAttachedWindow.getSession().setUaMode(defaultUaMode, true);
+            if (mHamburgerMenu != null) {
+                mHamburgerMenu.setUAMode(defaultUaMode);
+            }
+            if (mAttachedWindow.getSession() != null) {
+                mAttachedWindow.getSession().setUaMode(defaultUaMode, true);
+            }
```

Comment:
hmm why are we adding those null checks? Or putting it the other way around, why there where not there previously?